src/MerchantCenter/PriceBenchmarks.php

```diff
@@ -21,6 +22,196 @@
 class PriceBenchmarks implements ContainerAwareInterface, Service {
 
 	use ContainerAwareTrait;
+	use PluginHelper;
+
+	/**
+	 * Get the price benchmarks base data.
+	 *
+	 * @param array $args Query arguments.
+	 * @return array Mapped response data.
+	 */
+	private function get_price_benchmarks_base_data( $args ) {
```

Comment:
Now that we're merging data from all 3 Content API queries, we no longer need to separate "base data" from the merchant performance data. Let's fetch all 3 sources and pass them all to the `map_price_benchmarks_response()` method to format the data for inserting into the store DB.

Reply:
Addressed in https://github.com/woocommerce/google-listings-and-ads/pull/2935/commits/b52ed875410801918bee3071cd67b49cbb33d199